Sprint-1/implement/sum.test.js

```diff
@@ -13,24 +13,37 @@ const sum = require("./sum.js");
 // Given an empty array
 // When passed to the sum function
 // Then it should return 0
-test.todo("given an empty array, returns 0")
-
+//test.todo("given an empty array, returns 0")
+test (" ", ()=>{
```

Comment:
Could include a concise description in the form "Expect 0 when given an empty array" or even "Empty array" would be good.

This way, when a test failed, we can have more info about why the test failed.